src/chat/chat_panel.rs

```diff
@@ -764,10 +710,17 @@ impl ChatPanel {
             } | State::ModelSelectedWithEmptyChat { is_loading: false }
         ) {
             let store = scope.data.get_mut::<Store>().unwrap();
-            //store.send_chat_message(prompt.clone());
 
-            // MAE
-            store.send_agent_message(MaeAgent::Questioner, prompt.clone());
+            if let Some(agent_selected) = self
+                .prompt_input(id!(main_prompt_input))
+                .borrow()
+                .unwrap()
+                .agent_selected
```

Comment:
Made the selected agent public so the send logic remains here but probably it should be moved inside the prompt input.